src/Core/src/Handlers/GraphicsView/GraphicsViewHandler.Windows.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.UI.Xaml.Controls;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class GraphicsViewHandler : ViewHandler<IGraphicsView, UserControl>
```

Comment:
Must implement `NativeGraphicsView` in Microsoft.Maui.Graphics for Windows first.